src/Event/Subscriber/TimedPublishSubscriber.php

```diff
@@ -20,10 +23,22 @@ class TimedPublishSubscriber implements EventSubscriberInterface
     /** @var string */
     private $tablePrefix;
 
-    public function __construct(string $tablePrefix, EntityManagerInterface $entityManager)
+    public function __construct($tablePrefixes, ManagerRegistry $managerRegistry)
     {
-        $this->entityManager = $entityManager;
-        $this->tablePrefix = Str::ensureEndsWith($tablePrefix, '_');
+        $this->entityManager = $managerRegistry->getManager('default');
+        $this->tablePrefix = $this
+            ->setTablePrefixes($tablePrefixes, $managerRegistry)
+            ->getTablePrefix($this->entityManager);
+    }
+
+    /**
```

Comment:
Did you move this method to this position on purpose?

Reply:
That was by IDE formatter did, not me :)  ...reverted